config/routes.rb

```diff
@@ -26,111 +28,111 @@
   resources :ideas do
-    get '/start-conversation', on: :member, action: :start_conversation
-    get '/admin', on: :member, action: :admin
-    patch '/admin', on: :member, action: :admin_update
+    get "/start-conversation", on: :member, action: :start_conversation
+    get "/admin", on: :member, action: :admin
+    patch "/admin", on: :member, action: :admin_update
     patch :mark
-    patch '/up_score' => 'ideas#up_score'
-    patch '/down_score' => 'ideas#down_score'
-    get '/checklistitems' => 'ideas#checklistitems'
+    patch "/up_score", to: "ideas#up_score"
+    patch "/down_score", to: "ideas#down_score"
+    get "/checklistitems", to: "ideas#checklistitems"
   end
 
   devise_for :users,
     :skip => [:registrations, :sessions, :confirmations],
-    :controllers => { :omniauth_callbacks => "users/omniauth_callbacks", :passwords => 'users/passwords' }
+    :controllers => { :omniauth_callbacks => "users/omniauth_callbacks", :passwords => "users/passwords" }
 
   as :user do
     # Sessions
-    get    '/login'  => 'users/sessions#new', :as => :new_user_session
-    post   '/login'  => 'users/sessions#create'
-    get    '/user'   => 'users/sessions#show', :as => :user_session
-    delete '/logout' => 'users/sessions#destroy', :as => :destroy_user_session
+    get    "/login"  => "users/sessions#new", as: :new_user_session
+    post   "/login"  => "users/sessions#create"
+    get    "/user"   => "users/sessions#show", as: :user_session
+    delete "/logout", to: "users/sessions#destroy", as: :destroy_user_session
 
     # Registrations
-    controller 'users/registrations' do
-      get  '/signup', action: :new, as: :new_user_registration
-      post '/signup', action: :create, as: :user_registration
+    controller "users/registrations" do
+      get  "/signup", action: :new, as: :new_user_registration
+      post "/signup", action: :create, as: :user_registration
     end
 
-    get '/dashboard' => 'dashboard#index', as: :dashboard
-    get '/dashboard/:filter' => 'dashboard#index', as: :dashboard_filter
+    get "/dashboard", to: "dashboard#index", as: :dashboard
+    get "/dashboard/:filter", to: "dashboard#index", as: :dashboard_filter
 
     # settings
-    get    '/settings' => 'users#edit', as: :edit_user
-    patch  '/settings' => 'users/registrations#update'
-    get    '/settings/email' => 'users/registrations#edit_email', :as => :edit_user_email
-    get    '/settings/profile' => 'users/profiles#edit', :as => :edit_user_profile
-    get    '/settings/notifications' => "users/notifications#edit", :as => :settings_notifications
-    patch  '/settings/notifications' => "users/notifications#update"
-
-    namespace :users, path: 'user' do
+    get    "/settings", to: "users#edit", as: :edit_user
+    patch  "/settings", to: "users/registrations#update"
+    get    "/settings/email", to: "users/registrations#edit_email", as: :edit_user_email
+    get    "/settings/profile", to: "users/profiles#edit", as: :edit_user_profile
```

Comment:
Line is too long. [81/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.